clickhouse/clickhouse_test.go

```diff
@@ -0,0 +1,131 @@
+package clickhouse
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+type TestOrBench interface {
+	Helper()
+}
+
+func getTestConnection(t TestOrBench) (*Storage, error) {
+	t.Helper()
+
+	client, err := New(Config{
+		Host:   "127.0.0.1",
+		Port:   9001,
+		Engine: "Memory",
+		Table:  "test_table",
+		Clean:  true,
+	})
+
+	return client, err
+}
+
+func TestConnection(t *testing.T) {
```

Comment:
Can you add an underscore between "Test" and the what is being tested for all tests. Makes it easier to read